hieradata/bgo/common.yaml

```diff
@@ -102,7 +102,10 @@ profile::network::services::dns_records:
     "21.%{hiera('reverse_mgmt')}":     "%{::location}-status-01.%{hiera('domain_mgmt')}"
     "26.%{hiera('reverse_mgmt')}":     "%{::location}-nat-01.%{hiera('domain_mgmt')}"
     "31.%{hiera('reverse_mgmt')}":     "%{::location}-mq-01.%{hiera('domain_mgmt')}"
+    "34.%{hiera('reverse_mgmt')}":     "%{::location}-dns-01.%{hiera('domain_mgmt')}"
+    "35.%{hiera('reverse_mgmt')}":     "%{::location}-dns-02.%{hiera('domain_mgmt')}"
     "36.%{hiera('reverse_mgmt')}":     "%{::location}-image-01.%{hiera('domain_mgmt')}"
+    "39.%{hiera('reverse_mgmt')}":     "%{::location}-dns-03.%{hiera('domain_mgmt')}"
     "41.%{hiera('reverse_mgmt')}":     "%{::location}-db-01.%{hiera('domain_mgmt')}"
```

Comment:
Hva er grunnen til at PTR må være i under location mens A-records for de samme nodene er i common?

Reply:
Tanken har hele tiden vært å konsolidere dette i common/common.yaml, og jeg kan egentlig begynne på den jobben først som sist. Vi kan kontrollere hvor PTR-records brukes via andre mekanismer.